src/metabase/models/collection.clj

```diff
@@ -64,9 +64,8 @@
   (derive ::mi/write-policy.full-perms-for-perms-set))
 
 (def AuthorityLevel
-  "Schema for valid collection authority levels"
-  (s/maybe (s/enum "official")))
-
+  "Malli Schema for valid collection authority levels."
+  [:maybe [:enum "official"]])
```

Comment:
While we're at it I think this is worth refactoring to separate nilability from the rest of the schema.

So 
```suggestion
(def AuthorityLevel
  "Malli Schema for valid collection authority levels."
  [:enum "official"])
```

And move the `[:maybe ...]` to the endpoint level.

And is there any benefit in using `mu/with-api-error-message` to explain that it needs to be a valid authority_level?

Reply:
using `mu/with-api-error-message` will make the error message function use the provided string rather than the auto-generated one.

But I'll keep it as is in this case because:
- the auto-generate is good enough: it'll generate an error message: "nullable enum of official".
- we don't want to introduce new translated strings at this moment